From c5e731d1b73bd619c5d0d96bde4d6a91684a3840 Mon Sep 17 00:00:00 2001
From: Leonardo Fagundes Luz Serrano
 <Leonardo.FagundesLuzSerrano@windriver.com>
Date: Wed, 5 Jan 2022 14:25:43 -0300
Subject: [PATCH] Removing auto-update mechanism

According to the debian wiki, upstream/signing-key.asc and
watch files are used to check upstream for updates
by uscan (devscripts package).
Removing them as automatic updates are undesired.

Signed-off-by: Leonardo Fagundes Luz Serrano <Leonardo.FagundesLuzSerrano@windriver.com>
---
 debian/upstream/signing-key.asc | 94 ---------------------------------
 debian/watch                    |  4 --
 2 files changed, 98 deletions(-)
 delete mode 100644 debian/upstream/signing-key.asc
 delete mode 100644 debian/watch

diff --git a/debian/upstream/signing-key.asc b/debian/upstream/signing-key.asc
deleted file mode 100644
index 9379836..0000000
--- a/debian/upstream/signing-key.asc
+++ /dev/null
@@ -1,94 +0,0 @@
------BEGIN PGP PUBLIC KEY BLOCK-----
-
-mQINBF8VqoQBEACcjMco80+TaJU8JYCiOSPZb2a9pKSvz8I7wGqa+awcELWnTGTS
-1A66KneUNzEZlmXaL/OZ6n+KtMT+2V6hqBeG8+p/GbCDzr+ThwcdlUvQ9p15hiHV
-5IRPXgX15ltXB40+HEDQsj7iaZD14PmD/5QewfuGa/wEkGtPJjY+6+7ARE+JVdvG
-0Tzj9hlKlVJnJrEZELZh2+MFsRqtQPwx4h7JFZikLg1KHG8ewdo5zYjRLVrId334
-fu+cqvLa6dqXoM9adW5VREIyr4WJnyqCe+X1fc2K6ZVC+vonQw/6B0AIH6HYkah0
-jdwD1aMwLEFd3OzkqWn32Bxk75xviuxesOX08pAWoiDsbU7T2e+O0AXslBNVYDDf
-rnn0+OK8P0GzP391+OKC6fNN6I6Lfhs/+j5pNv5MzH2BdoRaO6bwnlQ8X1NuER3H
-/CZwrgN7XH01mRyNmWZBX/IJqNv5iDXqWcVQKqT+EKiy6XQqnFyQRRmzdj80/Ec3
-Xk55Ne73uMHExdVh5Rqv6/Pdn2lLT8IIsJiPS/NI9BamtvoyQC55M/ZgyMaTqQgr
-Bhf0U9qMn09gT8g+XmlSTPa7ldWCZn3nnemSQcSuCCmoHXOy+ZMQOtXAqnDhkLlr
-9gHAesWszMoFfGqDfgj23MgYRIHw0HRyuYnk2bP06GZsZbd9b1pgDfeyGwARAQAB
-tCVKacWZw60gRGVuZW1hcmsgPGpkZW5lbWFyQHJlZGhhdC5jb20+iQJUBBMBCAA+
-FiEERTtlMQWVVihVRxGZymi+gBAITJwFAl8VqoQCGwMFCQlmAYAFCwkIBwIGFQoJ
-CAsCBBYCAwECHgECF4AACgkQymi+gBAITJwAxg/6A/rc143J3rnHYVEzMbJR76MZ
-FGv+WQn0N42+d2I4zZiEYuZPopTpHahysDf6k3d9fZ0bv2mBNTzFvhlVMyqCKf9O
-1wPNnGdMCDhfCFjX1jkA9lW09XVpkKwqfMSV6nhEdlfabThzYyK9kWpRCD+wtTRo
-DMuZEdE1lPXql8PWyrSV0xxatISB1ULjpzq7muc9YtCyAZFho6W3n2EaPXhuNavt
-z1oke4HVPEuzMDXk3o6RMMjhrMa9KrfRfXdp9adfF1X43S2EaBBDI1uFn2w9iiVA
-Z2js4aZVxnEmiXO8XZRHvbgcDkRN5Wd/GsOKvZK7qoglPOMFpZ66UkneZsQv8fcp
-SIidkgpobvtfe28jMjZ//r//7k+9c8gHQPMcEQPXCjkELFpsUNBBXJ8WORQopp2u
-ZNigVpZ+nM/YcxYD4csDW6ieBCacEAvN3muzBqo01nk4IlbiZ6M5Ob+9aPAwXQjc
-9PQeB3qxlHfKwAvTcOk05dLJctMGBhnG8KBWnQbjIIF+1PC1Mdum8h88uZObS8qM
-yPtUtgxno1RCa3rhm1RPH8Wjr2Q1FWSMPNul8PHCagv6PZhts8a7i07hpizTiELk
-ny+lKaBzrTnge2tl4VbfzOOO1XHqifh1tKD4IyYe3BFeOvbc2w+EFwzpkcNp5Rse
-2uHgl2XL8NLIYZmb0bC5Ag0EXxWqhAEQALeQx73Ycb6gqDbJweVzY40IBkuG+4ES
-3zFcELCXDNTfityLTERMxJeuGuPi7IcoAYIjXl3eqUyiQgHsaodtL1y0u1x1BL1k
-tvx4D0ztskgflAFz+LB+wZSau3P0SfgZkNtBzxwtSuTVVnkC+DwT0XXGLDRBbyAo
-offF0NyurWRT7+uwx5cQ8z52M/w7lHeJywn8UonjjR+g+2IbsDsJYQuKuR0ColHf
-tf13O9G8xZJ+kiykMzQ1BHHinidFmTaSZwX8NKmJP67PQ5noMXBXv9YmC4Xd8IUe
-5t01A6Pjkaii8/SV08Pa3GiVx/Mw18EbkwpgudrtyCkiXnqnbB7/UYOEdO+zLRgl
-pRdBNDMF9gPWeoc0L8f9I0+ofuZ+tDTJnahKxwJdFWqfGfLy7i58Y1RVRUsQY88x
-lI9tUeaHabtXEA55c+RdCom5IN+tEzJgY00YkS5XcsZcfJvL5DStHlMow1l90lbN
-qxif0fFthZqYAuyJGlu5vWlRpFn/90sPo2GQse+VrFu2TF0wv5LsSKaj9oWzTCoP
-me6FOz2JCHWu+VnuKInnDN89nG4w1ajCiu9he3UywdxG3L8B2VLwfi7wT5sUnqqE
-O/SEdlxAJ3MvSu4h+En0qRAdjWkSWRy4wjSVOC+4YaxQtbsq18BnOI3zpkb21QFw
-NwpOdVbs6HWJABEBAAGJAjwEGAEIACYWIQRFO2UxBZVWKFVHEZnKaL6AEAhMnAUC
-XxWqhAIbDAUJCWYBgAAKCRDKaL6AEAhMnKc2D/9CfgTonQlHZR/qWDTxWg6WZTaM
-CvuMK7qFSOkOaXmilYNDf+MW4H9Dzg8GLZyhD9+9qeh/kMqLgiMwkPfqKlR8KhIc
-DwLiRP+QZyf+aHizFHTEbpSFcQ+tOZYQqdN+7Te0EA8WL0CefpQSgqY9iwj4R1DF
-b4cvoCe33A8nyJ6+FZoPZ5sZPUlUEgQ1tCjsgraquEkyZ31kbd7/6V17SPO6Oiwz
-HWlaOqJEofAqZC2S4B/qjsqiECLbRERAwJhHxVAREdtSWB29k4wqRjFIa0IwY+Nd
-DSSyc7DZ9CuPzDQcZE9BE9z1gqrI5J32kP4lG98ffL7w9qoLtk8ga9xt/Xk45iX+
-4E68csnHX60qW1/fRqO6aJMUPbrn9lGvCFwldOk88F60D19CQorpI4gtuk7tkCnK
-UCa/AOAkpZMuliQUWxLzqar2LPfu7n41qI5VHXP1ZUHYU5yboUgWeKnqWWLYXNoK
-vVI0teihvP/8tlz/5OHfUbiFzlYV8wVhs3faBJ0qRmTKiHmgRssDUM+LbJgXzxAp
-vTYaoamncuAz6ofaQDFGlg6q5L+4RUTAg7csabTNBLewJk0F5gbzlMf5iMqNpPND
-GZRlZGnYheeiWwZ5bk480/0B9owTBrn3tL6iwApeR62HQJ6SLEcdCDCVJkcMT2Hh
-5AegqaRBEaE3ko/icJkBogQ5NRH0EQQAjGELFNSzLN+6jjDCYU5H8m+Wq7j2qbvC
-AjUeJDZN+o0/5Xe6Df9MbajP3tkvVTswHdeuTznhKG8U1zdt+DDoY+jzPbiTToAK
-tTdZNXloRqVcNhyGLG+NAR4ClX/Zna7+9++S9fEm/Kr9wnAYSXyvKFtGb9qHq7lh
-Q6bDK/VC8ccAoJo2qbg6yQ2TexxLTgnDxfAqDUxvA/sHnRmyi714OjROVMq5Fd6n
-2MKz2ovOBLD2NVWlnphfoT3IfteK9AlCDlQA1/Ox2iAbE5hmf7uirPWWAC1WW8I0
-HMjaezD+GZryKXx13cYCyfvQ2yrOsk5YANZqFEhEcnxqv5DAvZ6t3VWIJmJscUbj
-vP3lSrn70rZ8YSK/2r96NwP9E+P4T109jeFgqb6QsSgyD8vf6al3GI0hFW0tZX2D
-+s38TQysLhdo+IEVprpyZz6MCBsdPmYZdXXOvAekDoManpyTZDW0ilFJttzJc0nk
-+s8sgCXF1v//88We7e2KuC/fl+hULSfA+5G3AfzUL/d1KmSp1KZQ/t+W/8TOsz+l
-Knq0OkRhbmllbCBWZWlsbGFyZCAoUmVkIEhhdCB3b3JrIGVtYWlsKSA8dmVpbGxh
-cmRAcmVkaGF0LmNvbT6IXgQTEQIAHgUCQ1KUiwIbAwYLCQgHAwIDFQIDAxYCAQIe
-AQIXgAAKCRBGBril3pW8HyXAAJ4vF6rgJUhes/FpycEOiO8q1UwOiACfTZ8Mlfnc
-BNcJA4NXsqIAupXFPL+0KERhbmllbCBWZWlsbGFyZCA8RGFuaWVsLlZlaWxsYXJk
-QHczLm9yZz6IXgQTEQIAFgUCOTUR9AQLCgQDAxUDAgMWAgECF4AAEgkQRga4pd6V
-vB8HZUdQRwABASjYAJ0bg3uZHA+/f974QtI0VE+0uKRpDACfTkTeyy9yyRzOOSzP
-coyLuvFDru65AQ0EOTUR/BAEAOq2lDkrxxTRuNotrVeNAURKfnsZFzZ01ykupok1
-LUzY77uYtFQwx65D8r2jPCC0Z4N470XQ4Yb/7VXJRyWeycpRP/+OWv9V2uX5dyZP
-LzQ1uYMtTxMIwBQUFIN++TG4HiKcMBEfvXOlLbADMWL9qNNziA8qu3C6SUKN6dht
-DiFHAAMFBADBsm+4DRNa8mqN6TPSoCvv4sLAIJWgEJvFnfwHbAETh+lstXcENkHH
-nCeJvX3xNTIQkUKP//FRRhxR0tgtuDA7JzlSIBFZ1mxCFTM6ebxXwvpCBe0a/Z6t
-wYK+MB6OsvttOBhdVHN69wm9hZ+2FppNTZ1SerXnpZQ7MU/uzL3zP4hOBBgRAgAG
-BQI5NRH8ABIJEEYGuKXelbwfB2VHUEcAAQE6wwCePL+uqpMxhi0dph9gu0hcNMuD
-mEgAoIbkOwWEzQJVRTO305bzlIYAOvUiuQENBFhAiSoBCADVKvDB+Sp3y3meH6Ab
-CQ33M83N1pRucyLgX5P5XSdIM7YKQzbW94DyS1N6roOeo0Sx8//TCa4zSWoXj3Al
-Xkwxon2gJxd0Ngi5oGVNh2u85QTtIkFHwNI9NHjibnkQFwbxTiZ86mNN/hGtCodO
-jtUIyFdp47HhUyXC3aJW/BGp+dC9N+pc/4ZiTzYtBM1Nhl96iqiAcNwyhXa4pHy4
-Zqoavad9wxcJ+Hk7Zn8VJqwye90xo82MPq6hVFVC2hDil3XCRsRIXDvFO74lwcA2
-EEJFoZj7tnSb7CAdBprPOIW7BMyxGgB3jPK3aBRCz1Z3JmDy+keQ0nC9a5jMLcb1
-vD1JABEBAAGJAWgEGBECAAkFAlhAiSoCGwIBKQkQRga4pd6VvB/AXSAEGQECAAYF
-AlhAiSoACgkQFViLJllr6l1EIQgAsxOREFVKxcv2xHOnob2c68vnboYT/KCX/ds4
-WJKSq568gYu+VZHBn33QFyPkYuV7MgoOER+1WLIDaF9FD1SNuPVVoDlW4P4satDP
-sy9EDZT5vI4FFu8hu/Togc5VF8mDzEEE07NAm96IsDWC7gMwhNT8su2G2HlQM56x
-tQLobmmHRpBUTcyMfns5SHkDemCLSXIm1RqeyeT3+pRPqQj8hSECJzzKMzJIlPwq
-sjWJ+3XE85MAgVtDcFanyfhFGlFHfCAxH+5nZ0ggbuvgIAIxgME4oHKs8izXeVTk
-56km4IldSkzCQmghT0FY7404ZDVcNyi90X00kaTqRV5XMqs0V+NBAJ9192PXgYld
-WZct8y5UKb2aTpViGwCdH3vSJrX4pcsGfPqdOrjoWwujhJy5AQ0EWECJaAEIALvA
-vlK+tI665d3OWl/qn07adFPrwKHwjAtio6bSzluxGsEiXAxJrNuqcmJQvUj6TBUX
-KDFDEIbka1IJiIcBI86p4TKq7p4syJ52yj5UWIJw1dYK3zGxUSkexBz0wOl6pwV5
-Icj5qDHE/LebSaGN1qcfPIT3ufMB/l608HM1SvglBQ+vAkbusePpvtSBY30zI849
-Ipy+8z4LcTcdJk3TKb52Qhc+AQt9eeeUzPyUgtE8j+rVAv0sIL/DiPraWdrNOZwC
-Tyvj58TUdNcquIKbuUw+kdVYldAt3DRSnq8ojYbW8aoe0ezpgjm26NtjHl7abdLr
-ct/r+E3NA/JREVBpjCEAEQEAAYhJBBgRAgAJBQJYQIloAhsMAAoJEEYGuKXelbwf
-5pwAn2uD0RpQ9xa7xPr1OYBmNV4HutvPAJ9Yj+vFte9+FNTqObFprqnf2Q+rdA==
-=wA/d
------END PGP PUBLIC KEY BLOCK-----
diff --git a/debian/watch b/debian/watch
deleted file mode 100644
index 4e8a107..0000000
--- a/debian/watch
+++ /dev/null
@@ -1,4 +0,0 @@
-version=3
-
-opts="uversionmangle=s/\.-rc/~rc/;s/\.+$//,pgpsigurlmangle=s/$/.asc/"  \
-    https://libvirt.org/sources/python/libvirt-python-([\d\.]+)(-rc\d)?\.tar\.gz
-- 
2.17.1

